internal/pkg/library/library.go

```diff
@@ -123,3 +123,24 @@ func SearchLibrary(ctx context.Context, c *client.Client, value string) error {
 
 	return nil
 }
+
+// PrepOutputImage will take a path to a file, and ensure its permission of
+// 777, and return it as *os.File.
+// TODO: maybe theres a better place for this?...
+func PrepOutputImage(path string) (*os.File, error) {
+	// Perms are 777 *prior* to umask in order to allow image to be
+	// executed with its leading shebang like a script
+	destFile, err := os.OpenFile(path, os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 0777)
+	if err != nil {
+		return nil, fmt.Errorf("while opening destination file: %s", err)
+	}
+
+	// Ensure the file permission is 777. If path already exist, os.OpenFile
+	// will **not** apply the specified permission of 0777.
+	err = destFile.Chmod(0777)
```

Comment:
Change this to `destFile.Chmod(0755)` ...

Reply:
Also ... this looks like it's going to override the users `umask` setting... Is that correct?